Dokumentationen/test/test_cases.adoc

```diff
@@ -27,15 +27,55 @@ a| * Datenbankeintrag für die Bestellung ist angelegt
 * Dozent befindet sich auf der Buchungsbestätigungseite
 
 |*Benötigte Daten*
-a| * Datum und Uhrzeit, Studiengang
+a| * Valides Datum, valide Uhrzeit und valider Studiengang
 * es muss ein Dozentenkonto geben 
```

Comment:
Hier auch nochmal an alle: Wäre es nicht sinnvoller den Punkt umzuformulieren? Sonst müsste er glaube eher zur Vorbedingung, oder nicht?

Reply:
Nö hätte ich nicht gesagt, weil es sind ja Daten und keine Vorbedingungen, die du erfüllen musst um an die entsprechende Stelle im Programm zu kommen, oder ?

Reply:
Inhaltlich stimme ich dir da absolut zu! Aber "es muss ein Dozentenkonto" klingt nach einer Bedingung. Wie wärs mit "Zugangsdaten des Dozentenkonto" als benötigte Daten?

Reply:
Guter Punkt was sagen die andren ?

Reply:
Also generell waren wir uns ja bei den benötigten Daten noch etwas unsicher... Meiner Meinung nach muss da alles hin was erzeugt werden muss um den Testdurchlauf zu starten. Wenn wir davon ausgehen, dass die Datenbank vor jedem Testdurchlauf leer ist (= leere Datenmenge) dann müssten wir durchaus erstmal ein Dozentenkonto in der Datenbank anlegen (was dann ein benötigtes Datum wäre). Also passt mir der jetzige Stand eigentlich besser, aber bin mir da eben auch unsicher. Vielleicht gibt es eine Formulierung die beides abdeckt? Oder wir warten erstmal auf Feedback und passen es dann an?

Reply:
Also ich hätte gesagt wir machen es so wie wir uns gemeinsam am ersten Meeting vereinbart haben und versuchen mal alles zu mergen. Dann wir es mit Herrn Anke abgesprochen und wir müssen dann nur noch eine Datei ändern und nicht vier Pull requests betreuen. Mein Postfach stirbt ... :)